File: Jaguar_api/src/main/java/ni/edu/uam/Jaguar_api/repository/EjercicioRepository.java
package ni.edu.uam.Jaguar_api.repository;

import ni.edu.uam.Jaguar_api.entity.Ejercicio;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface EjercicioRepository extends JpaRepository<Ejercicio, Long> {
}

File: Jaguar-Tracker/app/src/main/java/ni/edu/uam/jaguar_tracker/data/repository/EjercicioRepository.kt
package ni.edu.uam.jaguar_tracker.data.repository

import ni.edu.uam.jaguar_tracker.data.model.EjercicioDto
import ni.edu.uam.jaguar_tracker.data.remote.RetrofitClient

class EjercicioRepository {

    suspend fun obtenerEjercicios(): List<EjercicioDto> {
        return RetrofitClient.apiService.obtenerEjercicios()
    }
}